components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IMessageEncoder.cs

```diff
@@ -43,17 +43,19 @@ public interface IMessageEncoder {
         /// </summary>
         /// <param name="message"></param>
         /// <param name="maxMessageSize"></param>
+        /// <param name="useReversibleEncoding"></param>
```

Comment:
We should add descriptions of parameters for both `EncodeAsync()` and `EncodeBatchAsync()` methods.

Reply:
Added in #1694.